website/package.json

```diff
@@ -21,18 +21,25 @@
     "clsx": "^1.1.1",
     "mdx-mermaid": "^1.2.2",
     "mermaid": "^9.1.1",
-    "prism-react-renderer": "^1.3.1",
+    "prism-react-renderer": "^1.3.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-syntax-highlighter": "^15.5.0"
   },
   "devDependencies": {
     "@algolia/client-search": "^4.9.1",
     "@docusaurus/module-type-aliases": "2.0.0-beta.20",
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
+    "@mui/icons-material": "^5.8.2",
+    "@mui/material": "^5.8.2",
     "@tsconfig/docusaurus": "^1.0.5",
     "@types/react": "^17.0.0",
     "@types/react-syntax-highlighter": "^15.5.1",
     "docusaurus-plugin-typedoc": "^0.17.4",
+    "install": "^0.13.0",
+    "npm": "^8.12.1",
```

Comment:
hmm, that's interesting. Why do we need this explicitly?

Reply:
Likely a mistake... Will fix.